src/components/ContentIcon/ContentIconList.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <div>
+      <ContentIcon :content="content" v-for="content in contents"
+      :key="content.id"/>
+  </div>
+</template>
+<script>
+import axios from 'axios';
+import ContentIcon from './ContentIcon.vue';
+
+export default {
+  components: {
+    ContentIcon,
+  },
+  data() {
+    return {
+      contents: [],
+    };
+  },
+  name: 'ContentIconList',
+  mounted() {
+    axios.get('/api/contents', {
+      headers: {
+        'x-access-token': window.localStorage.getItem('x-access-token'),
+      },
+    }).then((res) => {
+      this.contents = res.data.content;
```

Comment:
api bize bir dizi gönderdiği için key ismi tekil olamamalı. `res.data.contents` şeklinde kullanabilmek için api'da gerekli değişiklikleri yapabilirsin.